src/Agent/NewRelic/Agent/Core/Attributes/AttributeDefinitionService.cs

```diff
@@ -126,7 +126,7 @@ public interface IAttributeDefinitions
         AttributeDefinition<object, object> GetCustomAttributeForSpan(string name);
         AttributeDefinition<object, object> GetCustomAttributeForTransaction(string name);
 
-        AttributeDefinition<string, string> GetLambdaAttribute(string name);
+        AttributeDefinition<object, object> GetLambdaAttribute(string name);
```

Comment:
Shouldn't this be AttributeDefinition<string, object>?

Reply:
I just got clarification that this refers to the type going in and the type going out, so the generic types are fine.